project/plugins.sbt

```diff
@@ -8,9 +8,6 @@ addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.2.9")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.3")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.10.0")
-addSbtPlugin(
-  "io.get-coursier" % "sbt-coursier" % coursier.util.Properties.version
```

Comment:
is there a place where this `coursier.util.Properties.version` is defined -- and no longer needed?

Reply:
https://github.com/scalameta/scalafmt/commit/f644b79585a0fceff82102d8f5d266508b09786c

Was probably coming from meta sbt-coursier, that's why in project/project/plugins.sbt version was fixed.